experimental/QuantumAnalogs/test/quantum_analogs.jl

```diff
@@ -0,0 +1,70 @@
+@testset "combinatorics/quantum_analogs.jl" begin
+  R,q = laurent_polynomial_ring(ZZ, "q")
+
+	# Test if everything specializes to usual stuff at q=1
```

Comment:
```suggestion
  # Test if everything specializes to usual stuff at q=1
```
there are some tabs left in this file